build.sbt

```diff
@@ -180,6 +180,9 @@ def formattingPreferences = {
     .setPreference(AlignParameters, true)
     .setPreference(AlignSingleLineCaseStatements, true)
     .setPreference(SpacesAroundMultiImports, true)
+    .setPreference(DanglingCloseParenthesis, Force)
+    .setPreference(DoubleIndentConstructorArguments, true)
```

Comment:
I think we want this one to be `false` to match the current style (or remove it, since that's the default).